### ISSUE TITLE
deprecated_command

### ISSUE THREAD

**Issue by StefanoD** [nushell/nushell#9865]
### Describe the bug

Hi,
I upgraded bushell from 0.82.0 to 0.83.0 with `brew upgrade` and after starting `nu`, I get following warning:
> Error: nu:: shell::deprecated_command
>
>  × Deprecated command let-env
>    ╭─[/Users/user/Library/Application Support/nushell/env.nu:46:1]
> 46 │ # Use nushell functions to define your right and left prompt
> 47 │ let-env PROMPT_COMMAND = {|| create_left_prompt }
>    · ───┬───
>    ·    ╰── 'let-env' is deprecated. Please use '$env.<environment variable> = ...' instead.
> 48 │ let-env PROMPT_COMMAND_RIGHT = {|| create_right_prompt }
>    ╰────

> Error: nu::shell::deprecated_command
>
>  × Deprecated command let-env
>     ╭─[/Users/user/Library/Application Support/nushell/config.nu:185:1]
> 185 │ # The default config record. This is where much of your global configuration is setup.
> 186 │ let-env config = {
>     · ───┬───
>     ·    ╰── 'let-env' is deprecated. Please use '$env.<environment variable> = ...' instead.
> 187 │   # true or false to enable or disable the welcome banner at startup
>     ╰────

### How to reproduce

1. Upgrade from nu shell 0.82.0 to 0.83.0
2. Start nu shell

### Expected behavior

There should be no warning

### Configuration

| key                | value                                                             |
| ------------------ | ----------------------------------------------------------------- |
| version            | 0.83.0                                                            |
| branch             |                                                                   |
| commit_hash        |                                                                   |
| build_os           | macos-aarch64                                                     |
| build_target       | aarch64-apple-darwin                                              |
| rust_version       | rustc 1.70.0 (90c541806 2023-05-31) (built from a source tarball) |
| cargo_version      | cargo 1.70.0 (ec8a8a0ca 2023-04-25)                               |
| build_time         | 2023-07-25 19:36:36 +00:00                                        |
| build_rust_channel | release                                                           |
| allocator          | standard                                                          |
| features           | dataframe, default, sqlite, trash, which, zip                     |
| installed_plugins  |                                                                   |

**Comment by amtoine** [comment]
@StefanoD 

the `let-env` has been removed in `0.83.0`.
you can use the following script
```nushell
# fix Nushell files after upgrading to 0.83.x
#
# # Examples
#     fix the config files
#     > fix
#
#     fix a single file
#     > fix /path/to/file.nu
#
#     fix all the `.nu` files in a directory recursively
#     > fix /path/to/directory/
export def fix [
    path?: path  # the path to a file or a directory with files to fix
]: nothing -> nothing {
    use std log

    def _fix [file: path]: nothing -> nothing {
        log debug $"fixing ($file)"
        open --raw $file | str replace --all --string "$env." "$env." | save --force $file
    }

    let path = $path | default $nu.default-config-dir

    if not ($path | path exists) {
        let span = metadata $path | get span
        error make {
            msg: $"(ansi red_bold)path_not_found(ansi reset)"
            label: {
                text: "no such file or directory"
                start: $span.start
                end: $span.end
            }
        }
    }

    match ($path | path type) {
        "file" => {
            _fix $path
            log info $"($path) has been fixed."
        },
        "dir" => {
            print $path
            ls ($path | path join "**" "*.nu") | each {|it| _fix $it.name}
            log info $"all files in ($path) have been fixed."
        },
        _ => { error make --unspanned {msg: $"(ansi red_bold)unknow internal issue(ansi reset)"} }
    }

    null
}
```
by running
```nushell
> fix
```
after pasting the script in the REPL :yum: 

let us know if it worked for you :blush:

**Comment by StefanoD** [comment]
Actually didn't work. Had to delete the complete config folder.
